homarr/docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: homarr_web_1
+      APP_PORT: 7575
+
+  web:
+    image: ghcr.io/homarr-labs/homarr:v1.30.0@sha256:6b686e5aa133a4a76d9a5558907b9be5f726b79e858750bf13dfe05e5380dc5a
+    restart: on-failure
+    environment:
+      - SECRET_ENCRYPTION_KEY=390314723a406d9680a370e85604e403655e9d38efff8de6f997ec45eb27c6cd

```

Use the generated app seed here.
```suggestion
      - SECRET_ENCRYPTION_KEY: ${APP_SEED}
```